spec/src/main/asciidoc/configexamples.asciidoc

```diff
@@ -119,3 +119,43 @@ public class InjectedConfigUsageSample {
 }
 ----
 
+=== Aggregate related properties into a single POJO Example
+
+When injecting a number of related configuration properties, it can be tedious to repeat the statement of `ConfigProperty` in scatter places. 
+Since they are related, it makes more sense to aggregate them into a single property class. 
+
+MicroProfile Config provides a way to look up a number of configuration properties starting with the same prefix using the `@ConfigProperties` annotation.
+Considering the following config source:
+
+[source, text]
+----
+config_ordinal = 120
+server.host = localhost
+server.port=9080
+server.endpoint=query
+----
+
+In order to retrieve the above properties in a single property class, you can use the `@ConfigProperties` annotation with the specified prefix.
+All of the fields specified in the class maps to the corresponding configuration properties. The field x maps to the configuration property <prefix>.x.
+
+[source, java]
+----
+
+@ConfigProperties("server")
+public interface ServerDetails {
+    String host; // the value of the configuration property server.host
+    int port;   // the value of the configuration property server.port
+    String endpoint; //the value of the configuration property server.endpoint
+    @ConfigProperty(name="oldServer.location")
+    String location; //the value of the configuration property oldServer.location
```

Comment:
These should be methods, since it is an interface.

Reply:
You were quite right. I started with class and then convert to interface, but forgot to update them.